tests/parsing/CliParsingTest.py

```diff
@@ -71,3 +71,8 @@ def test_check_conflicts(self):
         with self.assertRaisesRegex(SystemExit, '2') as cm:
             check_conflicts(sections)
             self.assertEqual(cm.exception.code, 2)
+
+        sections = parse_cli(arg_list=['--no-config', '--config', '.coafile'])
```

Comment:
Look at other tests in `test_check_conflicts`. None of them specify `.coafile`. It is not needed

Reply:
I think a .coafile is needed because if you run coala just with --no-config and --config an error of usage will be generated.

Reply:
"error of usage" is also generated for this test case:
https://github.com/raduklb/coala/blob/d4e3198b8fb5f3e9a356a71b171cf2e54c3e48c9/tests/parsing/CliParsingTest.py#L57?

Reply:
ok the `--config` expects one argument so another argument is needed after this.